modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookDisplayContext.java

```diff
@@ -210,6 +210,7 @@ private StyleBookEntry _getStyleFromThemeStyleBookEntry(long groupId) {
 		styleFromThemeStyleBookEntry.setGroupId(groupId);
 		styleFromThemeStyleBookEntry.setName(
 			LanguageUtil.get(_httpServletRequest, "styles-from-theme"));
+		styleFromThemeStyleBookEntry.setThemeId("classic_WAR_classictheme");
```

Comment:
We shouldn't be hardcoding the themeId. Handling the styleFromThemeStyleBookEntry requires a bit more work, since we need to have one for each theme or themeCSS CX that includes a frontend-token-definition.json file. This will mostly be handled by https://liferay.atlassian.net/browse/LPD-32807

For now get the themeID from the theme applied to the layoutSet.